cfme/tests/test_appliance.py

```diff
@@ -180,3 +181,28 @@ def test_keys_included(appliance, soft_assert):
         soft_assert(appliance.ssh_client.run_command(
             "test -e /var/www/miq/vmdb/certs/{}".format(k)).success,
             "{} was not included in the build".format(k))
+
+
+@pytest.mark.ignore_stream('5.9')
+def test_codename_in_log(configured_appliance):
+    log = '/var/www/miq/vmdb/log/evm.log'
+    configured_appliance.ssh_client.run_command('echo > {}'.format(log))
+    configured_appliance.ssh_client.run_command('appliance_console_cli --server=restart')
+
+    def codename_in_log():
+        r = configured_appliance.ssh_client.run_command(r'egrep "Codename: \w+$" {}'.format(log))
+        return r.success
+    wait_for(codename_in_log, fail_condition=False, timeout=30)
```

Comment:
If you're defining a function to call in `wait_for`, I'd request that you use the `wait_for_decorator` in order to decorate the function with your arguments for `wait_for` instead of having to call it separately.